themes/bootstrap3/templates/search/controls/showing.phtml

```diff
@@ -0,0 +1,30 @@
+<?
+  $transParams = [
+    '%%start%%' => $this->localizedNumber($this->results->getStartRecord()),
+    '%%end%%'   => $this->localizedNumber($this->results->getEndRecord()),
+    '%%total%%' => $this->localizedNumber($this->recordTotal),
+    '%%lookfor%%' => $this->escapeHtml($this->lookfor)
+  ];
+  $showingResults = '';
+?>
+<? if (!isset($this->skipTotalCount)): ?>
+  <? $showingResults = $this->translate('showing_results_of_html', $transParams); ?>
+<? else: ?>
+  <? $showingResults = $this->translate('showing_results_html', $transParams); ?>
+<? endif; ?>
+<? if (isset($this->overrideSearchHeading)): ?>
+  <? echo $this->layout()->srmessage = $showingResults; ?>
+  <?=$this->overrideSearchHeading?>
+<? elseif ($this->params->getSearchType() == 'basic'): ?>
+  <? if (!isset($this->skipTotalCount)): ?>
+    <? $showingResults = $this->translate('showing_results_of_for_html', $transParams); ?>
+  <? else: ?>
+    <? $showingResults = $this->translate('showing_results_for_html', $transParams); ?>
+  <? endif; ?>
+  <? $this->layout()->srmessage = $showingResults; ?>
+<? endif; ?>
```

Comment:
I interpreted the [original logic](https://github.com/vufind-org/vufind/blob/master/themes/bootstrap3/templates/search/results.phtml#L68) as overrideSearchHeading overriding the "for search: x" part of the header. Now that "Results 5-10 of 20, for search x" is tokenized, the logic got more complex.